charts/nginx-gateway-fabric/README.md

```diff
@@ -245,51 +252,56 @@ kubectl kustomize https://github.com/nginxinc/nginx-gateway-fabric/config/crd/ga
-| `service.externalTrafficPolicy`                         | The `externalTrafficPolicy` of the service. The value `Local` preserves the client source IP.                                                                                                            | Local                                                                                                           |
-| `service.annotations`                                   | The `annotations` of the NGINX Gateway Fabric service.                                                                                                                                                   | {}                                                                                                              |
-| `service.ports`                                         | A list of ports to expose through the NGINX Gateway Fabric service. Update it to match the listener ports from your Gateway resource. Follows the conventional Kubernetes yaml syntax for service ports. | [ port: 80, targetPort: 80, protocol: TCP, name: http; port: 443, targetPort: 443, protocol: TCP, name: https ] |
-| `metrics.disable`                                       | Disable exposing metrics in the Prometheus format.                                                                                                                                                       | false                                                                                                           |
-| `metrics.port`                                          | Set the port where the Prometheus metrics are exposed. Format: [1024 - 65535]                                                                                                                            | 9113                                                                                                            |
-| `metrics.secure`                                        | Enable serving metrics via https. By default metrics are served via http. Please note that this endpoint will be secured with a self-signed certificate.                                                 | false                                                                                                           |
-| `extraVolumes`                                          | Extra `volumes` for the NGINX Gateway Fabric pod.                                                                                                                                                        | []                                                                                                              |
+| Key | Type | Default | Description |
```

Comment:
Should we keep Description as the second column?

Reply:
I rearranged the columns using a custom template, I also added `` around the keys like we used to have.